ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Offering+HybridAdditionsTests.swift

```diff
@@ -68,11 +68,15 @@ class OfferingInfoHybridAdditionsTests: QuickSpec {
     }
 }
 
-private final class MockStoreProduct: StoreProductType, Sendable {
+// These are `enum`s, but for some reason Swift doesn't make them implicitly Sendable.
```

Comment:
Hmm I'm concerned of doing this without understanding the reason... As in, are we sure these are sendable in the public API?

Reply:
`enum`s without associated types are implicitly `Sendable`, just like `struct`s, because they don't have reference semantics. We could mark them as such in `purchases-ios`, but it's unnecessary. Since this is just a test type, I think it's easier to do this instead.